src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,77 @@
+import { Fragment } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+
+
+const activeTab = 'browse & explore'
+const styles = {
+  page: { padding: 20, marginTop: 15 }
+}
+
+const getSnapshot = id => {
+  return {
+    namespace: 'harry-potter',
+    name: 'Harry Potter',
+    created: '2020-01-07T18:25:28.340Z',
+    lastUpdated: '2020-01-07T18:25:28.340Z',
+    tags: {
+      itemsType: 'AttributeValue',
+      items: ['1000 Genomes', 'CMG', 'Open Access', 'Exome']
+    },
+    description: 'The boy who lived',
+    lowerName: 'harry potter',
+    lowerDescription: 'lower description',
+    project: {
+      id: '112',
+      name: 'CDC'
+    },
+    subjects: 10,
+    dataType: 'RNA Seq',
+    keepCollapsed: true,
+    locked: false,
+    files: 15,
+    fileSize: 4432
+  }
+}
+
+const DataBrowserDetails = routeParams => {
```

Comment:
For function components, our convention is to use the term `props`:
```suggestion
const DataBrowserDetails = props => {
```

Reply:
And since `id` seems to be the only property used from `routeParams`, we could destructure it like below:

```suggestion
const DataBrowserDetails = ({ id }) => {
```

and `id` will be available in the code below.

Reply:
Updated to use the ({id}) suggestion